src/lib/EventDecoder.js

```diff
@@ -1,10 +1,19 @@
-import ethAbi from 'ethereumjs-abi'
 import { addSignatureDataToAbi, getSignatureDataFromAbi } from './utils'
-import { remove0x, toBuffer, add0x, bufferToHex } from '@rsksmart/rsk-utils'
+import { remove0x, add0x, bufferToHex } from '@rsksmart/rsk-utils'
+import Web3Eth from 'web3-eth'
 
 function EventDecoder (abi) {
   abi = addSignatureDataToAbi(abi)
 
+  const eth = new Web3Eth('ws://localhost:8546');
+
+  const rawDecode = (types,data) =>{
+   const decoded = eth.abi.decodeParameters(types,data)
```

Comment:
We should use `web3-eth-abi` directly, there is no need to operate with the node to encode/decode ABI. See https://github.com/ChainSafe/web3.js/tree/1.x/packages/web3-eth-abi

> See https://github.com/ChainSafe/web3.js/blob/1.x/packages/web3-eth/src/index.js#L382